src/themes/dark.scss

```diff
@@ -3,7 +3,11 @@
   --wxc-text-color: #{$wxc-white-alpha-95};
   --wxc-meeting-border: 1px solid #{$wxc-white-alpha-30};
   --wxc-meeting-box-shadow: none;
-  
+
+  --wxc-banner-background: #{$wxc-gradation-dark-transparent};
+  --wxc-banner-text-color: #{$wxc-white-alpha-95};
+
+  // Icons
```

Comment:
this comment was removed on master, please remove it from your pr as well, thanks